02LessonTwo/in-class-project/index.html

```diff
@@ -15,20 +15,19 @@
   </div>
 
 
-    <table>
-        <tr>
-          <th><img src="https://www.barcelo.com/BarceloHotels/en_GB/Images/spa-swimming-pool-hotel-barcelo-sancti-patri-spa-resort21-2458.jpg" width="300px"></th>
-          <th><img src="https://www.barcelo.com/BarceloHotels/en_GB/Images/279-spa-29-hotel-barcelo-sancti-petri-spa-resort21-184279.jpg" width="300px"></th>
-          <th><img src="https://www.barcelo.com/BarceloHotels/en_GB/Images/279-room-suite-balinesa-2-hotel-barcelo-sancti-petri-spa-resort21-142251.jpg" width="300px"></th>
-        </tr>
-    </table>
-<div>
-<h3>
-<a href="https://www.barcelo.com/barcelohotels/en_gb/home-barcelo-hoteles.aspx">Home Page</a>
-<a href ="https://www.barcelo.com/barcelohotels/en_gb/mis-reservas/default.aspx">Bookings</a>
-<a href="https://www.barcelo.com/barcelohotels/en_gb/hotels/our-hotels.aspx">Hotels</a>
-</h3>
-</div>
+  <table border="1px">
+      <tr>
+        <th><img src="https://www.barcelo.com/BarceloHotels/en_GB/Images/spa-swimming-pool-hotel-barcelo-sancti-patri-spa-resort21-2458.jpg" width="300px"></th>
```

Comment:
**th** is used to declare the table header for that particular column. the rest of the rows or **tr** need to have **td** in which to add the content. take a look at this [example](http://www.w3schools.com/html/html_tables.asp)